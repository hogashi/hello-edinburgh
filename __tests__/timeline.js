import React from 'react';
import renderer from 'react-test-renderer';
import { act } from 'react-dom/test-utils';
import { configure, mount } from 'enzyme';
import Adaptor from 'enzyme-adapter-react-16';
configure({ adapter: new Adaptor() });

import Timeline from '../src/components/timeline';

describe('timeline', () => {
  it('render', () => {
    let component;
    act(() => {
      component = renderer.create(
        <Timeline />
      );
    });

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
