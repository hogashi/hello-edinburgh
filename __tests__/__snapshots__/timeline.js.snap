// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`timeline render 1`] = `
<div
  id="timeline"
>
  <p>
    reload in: 
    <span
      id="second"
    >
      60
    </span>
  </p>
  <button
    onClick={[Function]}
  >
    toggle
  </button>
  <div
    id="tweets"
  />
</div>
`;
